components/perl/B-COW/Makefile

```diff
@@ -11,18 +11,20 @@
 
 #
 # This file was automatically generated using the following command:
-#   $WS_TOOLS/perl-integrate-module Email::Valid
+#   $WS_TOOLS/perl-integrate-module B::COW
 #
 
 BUILD_STYLE = makemaker
 
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_PERL_MODULE =		B-COW
+COMPONENT_PERL_MODULE =		B::COW
 HUMAN_VERSION =			0.004
```

Comment:
The other new PR's also seem to have wrong COMPONENT_PERL_MODULE settings.

Reply:
> The other new PR's also seem to have wrong COMPONENT_PERL_MODULE settings.

Yes, I know.  I'm going through all of them...
This is simply because they were not created using the `perl-integrate-module` script.